hvtnFlow/resources/queries/flow/AnalysisPlanBase.sql

```diff
@@ -15,7 +15,11 @@
  */
 SELECT
 CONVERT('HVTN', 'VARCHAR') AS NETWORK,
-FCSAnalyses.FCSFile.Sample.Property.PROTOCOL AS PROTOCOL,
+-- "PROTOCOL" column collides with built-in sample set column and will not be visible in the sample set grid.
+-- Fortunately, bulk importing "PROTOCOL" will still persist the value so no data has been lost.
+-- Prefer using "_PROTOCOL" instead with "PROTOCOL" as the label so the value will be shown in the sample set grid.
+CAST(COALESCE(IFDEFINED(FCSAnalyses.FCSFile.Sample.Property.PROTOCOL),
```

Comment:
The argument order here doesn't match the comment.  This prefers PROTOCOL over _PROTOCOL.